infrastructure/src/main/java/org/corfudb/infrastructure/AbstractServer.java

```diff
@@ -16,6 +20,11 @@
     @Setter
     volatile boolean shutdown;
 
+    static final ExecutorService sharedExecutor = Executors
```

Comment:
Do we have any thoughts around any metrics on thread starvation/contention we want  to expose for further tuning this ?